tests/globalhelper/runhelper.go

```diff
@@ -64,6 +64,9 @@ func LaunchTests(testCaseName string, tcNameForReport string) error {
 		cmd.Stderr = outfile
 	}
 
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
```

Comment:
@theemj These two lines are already happening inside the logging files, we should stick to the logging files since they are not subject to change like our stdout.